packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -3,11 +3,11 @@ description: Flutter plugin for selecting images from the Android and iOS image
   library, and taking new pictures with the camera.
 repository: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
-version: 0.8.4
+version: 0.8.5
```

Comment:
similar to the google map PR, 0.8.4+1 is probably enough.

Reply:
Done.